vendor/github.com/moby/buildkit/client/graph.go

```diff
@@ -56,5 +56,6 @@ type SolveStatus struct {
 
 type SolveResponse struct {
 	// ExporterResponse is also used for CacheExporter
-	ExporterResponse map[string]string
+	ExporterResponse  map[string]string
+	PreferImageDigest bool
```

Comment:
Can't change the files in here. You can delete the config digest in resp (or add "prefer" key there) in build/build.go after build has completed.

Reply:
Got it, thanks; I opted for removing the config digest in that case. PTAL when you can, thanks!